corehq/tabs/tabclasses.py

```diff
@@ -717,7 +723,8 @@ def sidebar_items(self):
                     'show_in_dropdown': True,
                     'subpages': []
                 })
-            if BI_INTEGRATION_PREVIEW.enabled_for_request(self._request):
+            if (BI_INTEGRATION_PREVIEW.enabled_for_request(self._request)
+                    and self.can_view_odata_feed):
```

Comment:
`can_view_odata_feed` already checks for `BI_INTEGRATION_PREVIEW`, so I think you can drop that here and below.